### ISSUE TITLE
Support for NC 26

### ISSUE THREAD

**Issue by RunnerLexico** [nextcloud/twofactor_gateway#557]
Nextcloud 26 is out and the twofactor gateway is not compatible yet.

Does it work with Nextcloud 26?

Thanks and regards

**Comment by Cieniek** [comment]
Seems to work just fine on NC 26.

**Comment by vitormattos** [comment]
Yes, I think that work fine and only will be necessary to send a new PR bumping the Nextcloud version at info.xml file and verify if will be necessary to do changes at GitHub Actions to run using server with version 26

**Comment by nursoda** [comment]
Since NC27 is about to be released, it should be bumped to 27 right away. On my Arch instance and using NGINX, MariaDB, PHP 8.2 and SMS77, Authentication still works as expected.

**Comment by vitormattos** [comment]
Already was bumped to 27 here:
https://github.com/nextcloud/twofactor_gateway/commit/b3f29b5b9b704ea7a6f74e6b84d673298e415d1c

But this app haven't anymore a maintainer available to merge the PR's from community and release a new version.

Or you run the app from master branch doing all the necessary to build the app or wait by the next version.

**Comment by nursoda** [comment]
@nextcloud: I may maintain and release this app. However, I'd only manage it (and try to fix if something breaks), but I will not develop it further. Please contact me if you're interested. You'd also have to decide whether to move it out of nextcloud namespace. Ass user, I expect all apps within that namespace to be "officialy supported my nextcloud"!

**Comment by nursoda** [comment]
If you'd continue to maintain it, please do, that's far better than me mingling with it. Please reach out to @nextcloud Staff [via E-Mail or phone](https://nextcloud.com/team/) solve this deadlock situation. I assume no staff is reading here…

**Comment by florianschroen** [comment]
@ChristophWurst is there any progress in making @vitormattos the maintainer (ref #454), so keeping this app up to date can be done? thx

**Comment by ChristophWurst** [comment]
https://github.com/nextcloud/twofactor_gateway/issues/454 reopened.

**Comment by develroo** [comment]
Yeah, FWIW, now you get a warning this app is no longer compatible with 26.x. Needs testing and a bump, else it will be broken in future versions.



![Screenshot from 2023-08-24 11-44-01](https://github.com/nextcloud/twofactor_gateway/assets/7777993/21726c0a-a5ef-4a87-a09d-55907787091e)

**Comment by henmohr** [comment]
I think that  for backward compatibility you will need to hire some php devs @WeblinxIT 

Issue outdated @vitormattos and may be closed

**Comment by vitormattos** [comment]
Yes, very outdated, closing this.
